vassal-app/src/main/java/VASSAL/build/module/map/TextSaver.java

```diff
@@ -169,11 +161,11 @@ protected void writeMapAsText() {
 
   @Override
   public HelpFile getHelpFile() {
-    return HelpFile.getReferenceManualPage("Map.html", "TextCapture");
+    return HelpFile.getReferenceManualPage("Map.html", "TextCapture"); //NON-NLS
   }
 
   public static String getConfigureTypeName() {
-    return Resources.getString("Editor.TextCapture.component_type"); //$NON-NLS-1$
+    return Resources.getString("Editor.Saver.component_type"); //$NON-NLS-1$
```

Comment:
Type -> Editor.TextSaver.component_type?

Reply:
This is how I know I'm _actually_ getting old.